target-platform/tm4e-target.target

```diff
@@ -21,15 +21,28 @@
             <unit id="org.eclipse.jdt.junit5.runtime" version="0.0.0"/>
             <unit id="org.eclipse.ui.trace" version="0.0.0"/>
             <repository location="https://download.eclipse.org/eclipse/updates/4.23/"/>
+        </location>
+        <location includeDependencyDepth="direct" includeDependencyScopes="compile" includeSource="true" missingManifest="generate" type="Maven">
+            <dependencies>
+                <dependency>
+                    <groupId>com.google.code.gson</groupId>
+                    <artifactId>gson</artifactId>
+                    <version>2.9.0</version>
+                    <type>jar</type>
+                </dependency>
+            </dependencies>
+            <instructions><![CDATA[
+Bundle-Name:           Bundle derived from maven artifact ${mvnGroupId}:${mvnArtifactId}:${mvnVersion}
```

Comment:
Why is this needed? I just checked and https://repo1.maven.org/maven2/com/google/code/gson/gson/2.9.0/gson-2.9.0.jar is osgi artifact already?

Reply:
Then it is not. I didn't know.